microshift_storage/dynamic-provisioning-microshift.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="dynamic-provisioning-microshift"]
+= Dynamic provisioning for MicroShift
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-dynamic-provisioning
```

Comment:
Not sure if it's a hard rule, but usually we try to match the context and ID in an assembly. Consider updating them to be the same